packages/mgt-element/src/utils/Cache.ts

```diff
@@ -194,8 +209,8 @@ export class CacheService {
     if (Providers.globalProvider) {
       previousState = Providers.globalProvider.state;
     }
-
     Providers.onProviderUpdated(() => {
+      Providers.getCacheId(true);
```

Comment:
What is this call for? It looks like the cache will be cleared if the provider state is no longer signed in below, and we aren't using the cache id anywhere  in this method. Perhaps unnecessary?

Reply:
You're right, I don't think this is needed